.github/workflows/readme.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
     - run: python3 generate-readme.py
     - uses: googleapis/code-suggester@v3
       env:
-        ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        ACCESS_TOKEN: ${{ secrets.YOSHI_CODE_BOT_TOKEN }}
```

Comment:
This is the original key name https://github.com/googleapis/google-cloud-java/tree/main_202208/.github/workflows